teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>We usually use American English (AE) instead of British English (BE) in our notebooks - standardized (AE), normalised (BE) &gt; normalized (AE). I can proof-read at the end one more time to track down all BE terms.</p><p> </p><p>associated to &gt; associated with (2x)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Proofread once more.</p>